packages/web-api-scan-runner/src/runner/runner.ts

```diff
@@ -140,7 +140,7 @@ export class Runner {
         this.logger.logInfo(`Saving sarif results to Blobs...`);
         await this.pageScanRunReportService.saveSarifReport(reportId, JSON.stringify(sarifResults));
 
-        href = this.pageScanRunReportService.getBlobFilePath(reportId, format);
+        href = this.pageScanRunReportService.getBlobFilePath(reportId, `${reportId}.${format}`);
```

Comment:
this bug is about fixing the blob path in cosmos storage. currently it is being stored like this:
 "reports": [
        {
            "format": "sarif",
            "href": **"2019/9/20/23/sarif",**
            "reportId": "1e9dc004-5c80-6d40-f3c2-87730f63e0ae"
        }
    ],